src/devicemanager/devicemanager.h

```diff
@@ -59,7 +59,14 @@ class DeviceManager : public QAbstractListModel {
      *
      * @param linkConf
      */
-    Q_INVOKABLE void connectLink(LinkConfiguration* linkConf);
+    Q_INVOKABLE void connectLink(const LinkConfiguration* linkConf);
```

Comment:
Is there a reason this should not be passed by reference like the other?

Reply:
this function s called from the QML side, where QObjects are only used with pointers.